docker/README.md

```diff
@@ -0,0 +1,58 @@
+# `docker`
+
+This directory contains all of the repositories' dockerfiles as well as the [bake file](https://docs.docker.com/build/bake/)
+used to define this repository's docker build configuration.
+
+## Install Dependencies
+
+* `docker`: https://www.docker.com/get-started/
+* `docker-buildx`: https://github.com/docker/buildx?tab=readme-ov-file#installing
+
+## Building Locally
+
+To build any image in the bake file locally, use `docker bake`:
+
+```sh
+export TARGET="<target_name>"
```

Comment:
What's the target?

Reply:
Any target within the `docker-bake.hcl` file